src/makeInfinityAdder.js

```diff
@@ -1,12 +1,26 @@
 'use strict';
 
 /***
- *
  * @return {function}
  */
-
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(number) {
+    if (number === undefined) {
```

Comment:
Consider adding a check to ensure that `adder` is not called with more than one argument, as the current implementation assumes only one or zero arguments are passed.